lib/Base/Image.cpp

```diff
@@ -71,15 +71,6 @@ static llvm::cl::alias imageLayoutA("l",
                                     llvm::cl::desc("Alias for -image-layout"),
                                     llvm::cl::aliasopt(imageLayoutF),
                                     llvm::cl::cat(imageCat));
-
-bool useImagenetNormalization;
```

Comment:
Can you actually just move this to `ImageClassifier.cpp`, so the user doesn't have to specify the actual values of `imagenetNormMean`/`imagenetNormStd` on the command line? Then when setting `meanValues`/`stddevValues` inside `loadImagesAndPreprocess()` from `ImageClassifier.cpp` you can set the values correctly if `useImagenetNormalization`.